package.json

```diff
@@ -21,22 +21,30 @@
     "@babel/plugin-transform-object-assign": "^7.0.0",
     "@babel/preset-env": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
+    "@types/chai": "^4.2.10",
+    "@types/mocha": "^7.0.2",
```

Comment:
Are these used?

Reply:
These are used -- although not actively imported.   Since this repo uses type annotations in most JS files, my IDE (vscode) (and I expect others as well) can check types.   The repo is actually in very good shape type-wise, but there are a ton of errors in the test files b/c there are no type definitions for `mocha` and `chai`, which is why I added them to the repo.  Adding these dependencies silences very many false errors, and lets the type checking actually zoom in on real code problems to fix.
(These dependencies are just the type definitions from definitely-typed, no actual functional code).

Would you prefer that I remove them?

Reply:
Ah makes sense.  If it's helpful to you feel free to leave them.  They're not harming anything.